task_DEV_3/task_DEV_3/Program.cs

```diff
@@ -4,6 +4,38 @@ namespace task_DEV_3
 {
     class Program
     {
+        static void InputOfArrayElements(int[] numberArray, int arrayLength)
+        {
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine($"Enter an array element № {i}");
+                numberArray[i] = Convert.ToInt32(Console.ReadLine());
+            }
+        }
+        static void ExaminationNumberArray(int[] numberArray, int arrayLength)
+        {
+            bool isPrimeNumber;
+            for (int i = 0; i < arrayLength; i++)
+            {
+                isPrimeNumber = true;
+                for (int j = 2; j < numberArray[i]; j++)
+                {
+                    if (numberArray[i] % j == 0)
+                    {
+                        isPrimeNumber = false;
+                        break;
+                    }
+                }
+                if (isPrimeNumber)
+                {
+                    Console.WriteLine($"{ numberArray[i]} is prime numbr");
+                }
+                else
+                {
+                    Console.WriteLine($"{ numberArray[i]} is not prime number");
+                }
+            }
+        }
```

Comment:
Please add empty line between the methods.

Reply:
done